Replace Button test cases in favour of Buttons, extend test cases

lukewalczak opened the pull request:
Since there is no more `Button` block, I've refactored Button test cases into `Buttons` and extend them.

Update as well png related to `Buttons`

pinarol commented on test-cases/gutenberg/buttons.md:
```diff
@@ -0,0 +1,356 @@
+
+### Render custom background color
+
+-   Add `Buttons` block
+-   Save the post
+-   Open the same post on web
```
```suggestion
-   Open gutenberg web, add `Buttons` block
```

We don't need those first steps anymore, we can just render Buttons on mobile.

pinarol commented on test-cases/gutenberg/buttons.md:
```diff
@@ -0,0 +1,356 @@
+
+### Render custom text color
+
+-   Add `Buttons` block
+-   Save the post
+-   Open the same post on web
```
```suggestion
-   Open gutenberg web, add `Buttons` block
```

pinarol commented on test-cases/gutenberg/buttons.md:
```diff
@@ -0,0 +1,356 @@
+
+### Render gradient background color
+
+-   Add `Buttons` block
+-   Save the post
+-   Open the same post on web
```
```suggestion
-   Open gutenberg web, add `Buttons` block
```

pinarol commented on test-cases/gutenberg/buttons.md:
```diff
@@ -0,0 +1,356 @@
+
+### Fallback to default colors in case theme colors are used
+
+-   Add `Buttons` block
+-   Save the post
+-   Open the same post on web
```
-   Open gutenberg web, add `Buttons` block

pinarol commented on test-cases/gutenberg/buttons.md:
```diff
@@ -0,0 +1,356 @@
+
+### Check if selection / caret color matches font color
+
+-   Add `Buttons` block
+-   Save the post
+-   Open the same post on web
```
```suggestion
-   Open gutenberg web, add `Buttons` block
```

lukewalczak commented on test-cases/gutenberg/buttons.md:
```diff
@@ -0,0 +1,356 @@
+### Add another Button inside Buttons using inserter
+
+-   Add `Buttons` block
+-   Press inserter
```
Why do we need that step?

pinarol replied:
Hmm I think I missed the fact that it is automatically focused on a single Button when you create Buttons.

lukewalczak commented:
I refactored test cases according to your comments and merge some test cases into one. I also removed the not about known issue with Button since with current steps it won't be visible (we are creating Buttons on web instead of mobile).
